samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java

```diff
@@ -0,0 +1,25 @@
+package com.hexascribe.vertexai.controller;
+
+import com.hexascribe.vertexai.service.VertexService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/vertex")
+public class VertexController {
+
+    @Autowired
```

Comment:
it's usually recommended to inject dependencies through constructor as it reduces the dependency from framework when doing unit tests.

Reply:
as it's just a sample I wasn't paying attention to it, but I totally agree! Made the adjustments